Hestia/Hestia/frontend/resources/Globals.cs

```diff
@@ -18,8 +18,12 @@ public static class Globals
         public static String ServerName { get; set; }
         public static int Port { get; set; }
         public static String IP { get; set; }
+
+        public static String userName { get; set; }
+
         public static UIColor defaultLightGray { get; set; }
 
+
         public static ServerInteractor serverToAddDeviceTo { get; set; }
```

Comment:
These need to start with capital letters if they are meant to be properties.

Reply:
Zino, I only added the userName... this is the problem?